python_gdeploy/actions/install_gluster.py

```diff
@@ -0,0 +1,46 @@
+import ConfigParser
+
+from python_gdeploy.wrapper import gdeploy_features as gf
+from python_gdeploy.wrapper.gdeploy_wrapper import cook_gdeploy_config
+from python_gdeploy.wrapper.gdeploy_wrapper import invoke_gdeploy
+
+
+GLUSTERFS_PACKAGES = [
+    "glusterfs",
+    "glusterfs-server",
+    "glusterfs-cli",
+    "glusterfs-libs",
+    "glusterfs-client-xlators",
+    "glusterfs-api",
+    "glusterfs-fuse"
+]
+
+
+def get_glusterfs_repo():
+    config = ConfigParser.SafeConfigParser()
+    config.read('/etc/python-gdeploy/python-gdeploy.conf')
+    return config.get("python-gdeploy", "glusterfs_repo")
+
+
+def install_gluster_packages(host_list,
+                             glusterfs_packages=None,
+                             glusterfs_repo=None,
```

Comment:
the provisioning wrapper calls this method and it has to pass glusterfs repos and glusterfs package list. I feel we need to maintain these details in master definition file, under the import cluster flow. 

Correct me if my understanding is wrong